docs/src/pages/demos/progress/progress.md

```diff
@@ -48,9 +48,9 @@ For example, a refresh operation should display either a refresh bar or an activ
 
 ## Delaying appearance
 
-There is a [school of thought](http://www.nngroup.com/articles/response-times-3-important-limits/)
-that feedback after user operations isn't necessary for about a second.
-To implement this, you can take advantage of the [Fade](/api/fade)
-transition `enterDelay` prop to delay the appearance of the progress indicators.
+There is [3 important limits](http://www.nngroup.com/articles/response-times-3-important-limits/) to know around reponse time.
+The ripple effect of the `ButtonBase` component will make sure the user feel that the system is reacting instantaneously.
```

Comment:
/will make sure/ensures that/ ("will make sure" is not incorrect, but "ensures that" is preferable here in terms of word choice and change of tense)
/feel/feels/